### PR TITLE
Mssql date time literal support

### PR DESCRIPTION
**Pull request by jdelporte**
Formatted date/datetime cannot be used in date operation in SqlServer.
This PR fixes it

### CODE REVIEW

**Review by akimd — APPROVED** [review]
LGTM.
But fixes should be squashed.

File: lib/arel_extensions/visitors/mssql.rb [hunk 1]
```diff
@@ -117,7 +118,7 @@ def quote value, column = nil
           when Numeric, ActiveSupport::Duration
             value.to_s
           when Date, Time
-            "'#{quoted_date(value)}'"
+            "#{quoted_date(value)}"
```
Comment (akimd) [inline lib/arel_extensions/visitors/mssql.rb:121]: Do we still need the interpolation?

Comment (jdelporte) [reply lib/arel_extensions/visitors/mssql.rb:121]: nope

**Review by akimd — APPROVED** [review]
👍

**Pull request merged**